Functions.Templates/Templates/DurableFunctionsOrchestrator-JavaScript/index.js

```diff
@@ -0,0 +1,19 @@
+﻿/*
+ * Before running this sample, please:
+ *  - create a Durable Activity function (default name is "Hello")
+ *  - run 'npm install durable-functions' from the wwwroot folder of your function app in Kudu
```

Comment:
I wonder if we should include this npm comments in the other Durable JS templates as well.